### ISSUE TITLE
Anzeige Akkukapazität

### ISSUE THREAD

**Issue by StevieC121176** [solectrus/solectrus#2059]
Wäre es möglich die aktuelle Akkukapazität durch z. Bsp. einen Tooltip anzuzeigen.

Beispiel Mauszeiger auf die % Anzeige Batterie -> 11,52 von 12,8kWh bei 90% Akku

 
![IMG_0786](https://github.com/solectrus/solectrus/assets/113791299/b1d52187-6c06-4433-b42b-b5fea8c0435b)
![IMG_0787](https://github.com/solectrus/solectrus/assets/113791299/d558c468-ea28-4767-9d1f-41cb028de747)

**Comment by ledermann** [comment]
Ja, wäre prinzipiell machbar. Derzeitig kennt SOLECTRUS die Kapazität des Speichers jedoch nicht, d.h. die müsste erst noch erfasst werden (am besten per `.env`).

**Comment by StevieC121176** [comment]
EVCC stellt diesen Wert per MQTT zur Verfügung. Ich denke auch es wäre analog der anderen Werte in der *.env am besten aufgehoben.
